src/components/20-molecules/list/README.md

```diff
@@ -0,0 +1,76 @@
+# List
+
+List in various variants. Standardizes the use of `ol` and `ul` tags.
+
+## Usage
+
+**Important:** If this component needs to run in Internet Explorer 11, [you need to use our polyfill](https://github.com/axa-ch/patterns-library/tree/develop/src/components/05-utils/polyfill).
+
+```bash
+npm install @axa-ch/list
+```
+
+```js
+import '@axa-ch/list';
+...
+<axa-list></axa-list>
+```
+
+### React
+
```

Comment:
Is this needed? I always deleted this 🤷‍♂️

Reply:
Yeah you're right!

Reply:
Remove that code-creation from create-amo script or create issue for that.